htcanalyze/main.py

```diff
@@ -514,12 +517,11 @@ def print_results(htcanalyze: HTCAnalyze,
     # Allow this to happen
     if results is None:
         sys.exit(0)
-    # convert result to list, if given as dict, else copy
-    processed_data_list = [results] if isinstance(results, dict)\
-        else results[:]
+    # convert result to processed data list, if given as dict, else copy
+    proc_data_list = [results] if isinstance(results, dict) else results[:]
```

Comment:
You don't need `results[:]`, because it's the same as `results`

Reply:
Mh results[:] is actually a copy from results, whereas results would just be a reference. However this does not matter here, but I wanted to make sure results is not getting changed.

Reply:
Lol just found out that list has a .copy function, I will use this instead of cycling, so it is more obvious

Reply:
yeah .copy() is better, because "best practices" for Python programming is to be explicit :)